GameData/RealismOverhaul/RO_SAFConfig.cfg

```diff
@@ -0,0 +1,84 @@
+/// Example config - Replace 'MyPartWithSAFModule' with your part name
+/// @PART[MyPartWithSAFModule]:FOR[RealismOverhaul]
+/// {
+/// 	%ROSAFRescale = 1.81 //Make everything 1.81 times bigger
+/// }
+
+@PART[*]:HAS[@MODULE[ModuleSimpleAdjustableFairing]:HAS[#ROSAFRescale]]:AFTER[RealismOverhaul]
+{
+	@MODULE[ModuleSimpleAdjustableFairing],*
+	{
+		&scale = 1
+		@scale *= #$/ROSAFRescale$
+		@segmentLength *= #$/ROSAFRescale$
+		@shieldingCenter[*] *= #$/ROSAFRescale$
+		@shieldingBaseRadius *= #$/ROSAFRescale$
+		@editorOpenOffset[*] *= #$/ROSAFRescale$
+
+		@WALL_BASE
+		{
+			@CoM[*] *= #$/ROSAFRescale$
+			@rootOffset[*] *= #$/ROSAFRescale$
+		}
+
+		@WALL
+		{
+			@CoM[*] *= #$/ROSAFRescale$
+			@rootOffset[*] *= #$/ROSAFRescale$
+		}
+
+		@CONE
+		{
+			@CoM[*] *= #$/ROSAFRescale$
+			@rootOffset[*] *= #$/ROSAFRescale$
+		}
+
+		@CAP
+		{
+			@CoM[*] *= #$/ROSAFRescale$
+			@rootOffset[*] *= #$/ROSAFRescale$
+		}
+	}
+
+	@MODULE[ModuleB9PartSwitch],*
+	{
+		@SUBTYPE,*
+		{
+			@MODULE:HAS[@IDENTIFIER[ModuleSimpleAdjustableFairing]]
+			{
+				@DATA
+				{
+					@WALL_BASE
+					{
+						@CoM[*] *= #$/ROSAFRescale$
+						@rootOffset[*] *= #$/ROSAFRescale$
+					}
+
+					@WALL
+					{
+						@CoM[*] *= #$/ROSAFRescale$
+						@rootOffset[*] *= #$/ROSAFRescale$
+					}
+
+					@CONE
+					{
+						@CoM[*] *= #$/ROSAFRescale$
+						@rootOffset[*] *= #$/ROSAFRescale$
+					}
+
+					@CAP
+					{
+						@CoM[*] *= #$/ROSAFRescale$
+						@rootOffset[*] *= #$/ROSAFRescale$
+					}
+				}
+			}
+		}
+	}
+}
+
+// Delete the rescale tag
+@PART[*]:HAS[@MODULE[ModuleSimpleAdjustableFairing]]:FOR[zzzTagCleanup]
+{
+	!ROSAFRescale = null
+}
```

Comment:
`:HAS[#ROSAFRescale]` not the `@MODULE`.
Also add newline before EOF.

Reply:
Fixed